.github/workflows/code_style.yml

```diff
@@ -1,5 +1,9 @@
 name: Code Style
-on: [push, pull_request]
+on:
+  push:
+    paths: '**.java'
```

Comment:
The reason we constrained it to java files is because in cuda_plugin we have separate code style check

Reply:
In last update, added exclusion for `modules/cuda_plugin/**`